composer.json

```diff
@@ -6,30 +6,40 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^9.3",
-        "drupal/sparql_entity_storage": "1.0.0-alpha10",
-        "easyrdf/easyrdf": "1.0.0 as 0.9.1",
-        "php": ">=7.4"
+        "drupal/core": "^9.4",
+        "drupal/sparql_entity_storage": "^2.0.0-alpha9",
+        "sweetrdf/easyrdf": "^1.7.1",
+        "php": ">=8.0"
     },
     "require-dev": {
+        "behat/mink-selenium2-driver": "^1.5",
         "composer/installers": "^1.5",
-        "consolidation/robo": "^3.0",
+        "consolidation/robo": "^4.0",
         "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^9.3",
-        "drupal/core-dev": "^9.3",
-        "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~10.0",
-        "fabpot/goutte": "~3.3",
+        "drupal/core-composer-scaffold": "^9.4",
+        "drupal/core-utility": "^9.4",
+        "drupal/drupal-extension": "dev-master",
+        "drush/drush": "^11.1",
+        "guzzlehttp/guzzle": "^7.0",
+        "masterminds/html5": "^2.7.6",
+        "mikey179/vfsstream": "^1.6.10",
         "nikic/php-parser": "^4.12.0",
-        "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "^2.0",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "openeuropa/behat-transformation-context": "dev-master",
```

Comment:
I wouldn't add it because we will not release this component using master branch for `behat-transformation-context`. After the update is done for all the components we should release them and update everywhere we use the default branch.